generator/routing_generator.cpp

```diff
@@ -234,7 +234,7 @@ void BuildCrossRoutingIndex(string const & baseDir, string const & countryName,
   routing::CrossRoutingContextWriter crossContext;
   FindCrossNodes(nodeData, osm2ft, countries, countryName, crossContext);
 
-  string const mwmRoutingPath = localFile.GetPath(MapOptions::CarRouting);
+  string const mwmRoutingPath = localFile.GetPath(MapOptions::CarRouting) + ROUTING_FILE_EXTENSION;
```

Comment:
Why? GetPath(MapOptions::CarRouting) will return you a path with ROUTING_FILE_EXTENSION at the end.